src/language/CodeInspection.js

```diff
@@ -199,6 +199,21 @@ define(function (require, exports, module) {
     }
 
     /**
+     * Returns an array of the IDs of providers registered for a specific language
+     *
+     * @param {!string} languageId
+     * @return {?Array.<string>} Names of registered providers.
+     */
+    function getProviderIDsForLanguage(languageId) {
+        if (!_providers[languageId]) {
+            return null;
```

Comment:
I vote for empty array here, too.
It's easier to use with things like `.map`, `.filter`, and so on.